src/main/java/org/openstreetmap/atlas/checks/distributed/AtlasDataSource.java

```diff
@@ -38,7 +38,7 @@
 {
     private static final long serialVersionUID = -6407331424906155431L;
     private Atlas atlas = null;
-    private final Logger logger = LoggerFactory.getLogger(AtlasDataSource.class);
+    private final transient Logger logger = LoggerFactory.getLogger(AtlasDataSource.class);
```

Comment:
This logger variable will deserialize to null and could cause issues.

Reply:
I will assign null to it and add a getter method to set and get the LoggerFactory.getLogger(AtlasDataSource.class).